br/pkg/backup/client.go

```diff
@@ -534,7 +534,9 @@ func (bc *Client) BackupRanges(
 	progressCallBack func(ProgressUnit),
 ) error {
 	init := time.Now()
-	defer log.Info("Backup Ranges", zap.Duration("take", time.Since(init)))
+	defer func() {
+		log.Info("Backup Ranges", zap.Duration("take", time.Since(init)))
```

Comment:
Good ~
Maybe we can print more information about `len(ranges)`.

Reply:
Thanks! 
And yes, later I will have a follow-up pull request for modifying some printed log messages, which will also include the `len(ranges)` part.